### ISSUE TITLE
ProcessModelSection.AutoConfig

### ISSUE THREAD

**Issue by BellBat** [dotnet/dotnet-api-docs#3974]
Details for how the AutoConfig property actually works are not documented.
e.g. When autoconfig is true what properties get set to what values.

Example: I am very confident maxconnections get set to int.maxvalue, because I checked the reference source for 4.8 and recall from 4.7, but I do not remember when it changed or all the other properties.

https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Web.Configuration/ProcessModelSection.xml

**Comment by carlossanlop** [comment]
@gewarren is this an ASP.NET type? I ask so I know who to tag for triage, or maybe check if this namespace should be moved to the AspNetApiDocs repo.

**Comment by gewarren** [comment]
Yes, this looks like an ASP.NET type. I don't know the history of why these APIs aren't in the ASP.NET reference, but I believe that @scottaddie is the owner for them in this repo.

**Comment by carlossanlop** [comment]
Thanks. I suppose there might be a good reason to have the System.Web API live here, so I'll go ahead and create a label for it.
